<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口终端</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Widget
{
	border-radius: 10px;
	border:5px solid rgb(255, 213, 129);
	background-color: rgb(107, 80, 64);
}

QComboBox
{
	color: rgb(255, 255, 255);
	background-color: rgb(165, 165, 165);
	border-bottom: 2px solid black;
	border-radius: 3px;
}

QComboBox QAbstractItemView {
    outline: 0px solid gray;   
    border: 1px solid gray;   
    color: white;
    background-color: rgb(165, 165, 165);   
    selection-background-color: rgb(65, 65, 65);   
}

QComboBox QAbstractItemView::item:hover {
    color: white;
    background-color: rgb(65, 65, 65);   
}

QComboBox QAbstractItemView::item:selected {
    color: white;
    background-color: rgb(48, 48, 48);
	outline:4px black;
}

QToolButton:hover
{
	border-radius: 2px;
	border:5px rgb(255, 255, 255);
}

QPlainTextEdit
{
	background-color: rgb(0, 0, 0);
	color: rgb(255, 255, 255);
}

QToolButton#refreshPort
{
	background-color: rgb(112, 112, 112);
}

QToolButton#cleanButton
{
	background-color: rgb(112, 112, 112);	
}

QToolButton#setButton
{
	background-color: rgb(112, 112, 112);	
}

QToolButton#fileButton
{
	background-color: rgb(112, 112, 112);	
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <item>
      <widget class="QToolButton" name="refreshPort">
       <property name="toolTip">
        <string>刷新端口</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/refresh.png</normaloff>:/image/refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="run">
       <property name="toolTip">
        <string>start</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#run
{
	background-color: rgb(0, 175, 0);
	border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/start.png</normaloff>:/image/start.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cleanButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/clean.png</normaloff>:/image/clean.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="setButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/set.png</normaloff>:/image/set.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="port">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pbs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>波特率</string>
       </property>
       <property name="currentText">
        <string>115200</string>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="data">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>数据位</string>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>停止位</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="parity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>奇偶校验</string>
       </property>
       <item>
        <property name="text">
         <string>NoParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EvenParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OddParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SpaceParity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MarkParity</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
     <item>
      <widget class="QPlainTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="preview"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="transTextButton">
       <property name="text">
        <string>发送文本文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>预览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fileButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/image/file.png</normaloff>:/image/file.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>250</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
